tools/bazel.rc

```diff
@@ -1,5 +1,8 @@
 # TODO(#2897): move to %workspace%/.bazelrc
 
+# Remote Build Execution requires a strong hash function, such as SHA256.
+startup --host_jvm_args=-Dbazel.DigestFunction=SHA256
```

Comment:
Should this be put into the remote config only? This might intersect weirdly with user config overrides.

Reply:
I didn't see a reason to limit it, but it's fine.